post.hbs

```diff
@@ -49,13 +49,17 @@ into the {body} of the default.hbs template --}}
             <figure class="post-full-image">
                 {{!-- This is a responsive image, it loads different sizes depending on device
                 https://medium.freecodecamp.org/a-guide-to-responsive-images-with-ready-to-use-templates-c400bd65c433 --}}
-                <img srcset="{{img_url feature_image size="s"}} 300w,
-                            {{img_url feature_image size="m"}} 600w,
-                            {{img_url feature_image size="l"}} 1000w,
-                            {{img_url feature_image size="xl"}} 2000w" sizes="(max-width: 800px) 400px,
-                            (max-width: 1170px) 700px,
-                            1400px" src="{{img_url feature_image size="xl"}}" alt="{{title}}"
-                    onerror="this.style.display='none'" />
+                <picture>
+                    <source media="(max-width: 700px)" sizes="1px"
+                        srcset="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7 1w" />
```

Comment:
```suggestion
```
If this source is missing, nothing gets loaded for small screens which is what we want.

Reply:
seems like it will load the <img src which is our xl image

Reply:
Thanks, I understand now - it worked locally only because the src was missing.  Now that you've fixed that, it defaults to it.